openmc/filter.py

```diff
@@ -2021,6 +2054,19 @@ def y(self, y):
     def bins(self, bins):
         raise RuntimeError('EnergyFunctionFilters have no bins.')
 
+    @interpolation.setter
+    def interpolation(self, val):
+        cv.check_type('interpolation', val, str)
```

Comment:
Would it be worth adding a doc string here with the val argument defined as I believe this is the main point of interaction for the user

Reply:
We don't normally do this for properties -- they are documented in the class docstring itself